docs/development/making_pr.rst

```diff
@@ -0,0 +1,92 @@
+Making a PR to SecureDrop
+=============================
+
+Forking and Cloning the Project
+---------------------------------
+
+1. Fork SecureDrop on GitHub from the Main Repository to your own profile.
+
+2. Clone the forked repository.
+
+.. code:: sh
+
+  git clone https://github.com/<your-username>/securedrop.git
+  cd securedrop
+
+3. Add the Main Repository as an upstream remote.
+
+.. code:: sh
+
+  git remote add upstream https://github.com/freedomofpress/securedrop.git
+
+
+Make Your Changes and Push to the Fork
+----------------------------------------------
+
+Create a branch
+~~~~~~~~~~~~~~~~~~
+
+Create a branch on which you make your changes.
+
+.. code:: sh
+
+  git checkout -B change-one
+
+
+Make Your Changes and Commit
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Now enter the directory of your fork amd make changes as you wish.
+Run tests for the changes you have made.
+
+If you create a new file, remember to add it with :code:`git add`.
+
+.. code:: sh
+
+  git add <new-file>
+
+Commit your changes, adding a description of what was added. If you’re not
+used to Git, the simplest way is to commit all modified files and add a
+description message of your changes in a single command like this:
+
+.. code:: sh
+
+  git commit -a -m "<Description of changes made>"
+
+Merge Your Changes to develop branch
```

Comment:
So this "Merge Your Changes to develop branch" section isn't actually necessary - it's perfectly fine to request a pull request from a branch on your remote called e.g. `"docs-pr-update"`.